.php-cs-fixer.php

```diff
@@ -27,6 +27,7 @@
     'concat_space' => ['spacing' => 'one'],
     'visibility_required' => false,
     'no_null_property_initialization' => false,
+    'static_lambda' => false,
```

Comment:
this seems to be improved in the latest phpcs-fixer version, so that it added the static function to pest tests as well.

That made them fail with:
  Cannot bind an instance to a static closure

  at vendor/pestphp/pest/src/Factories/TestCaseFactory.php:151
    147▕             $proxies->proxy($this);
    148▕             $chains->chain($this);
    149▕
    150▕             /* @phpstan-ignore-next-line */
  ➜ 151▕             return call_user_func(Closure::bind($factoryTest, $this, get_class($this)), ...func_get_args());
    152▕         };
    153▕
    154▕         $className = $this->makeClassFromFilename($this->filename);
    155▕
    
(obviously)

So i had to disable that rule, cause otherwise phpcs fixer would add the statics that would make pest fail